src/dcos_e2e/node.py

```diff
@@ -226,24 +237,28 @@ def send_file(
             local_path: The path on the host of the file to send.
             remote_path: The path on the node to place the file.
             user: The name of the remote user to send the file via
-                secure copy.
+                secure copy. If `None` the ``default_ssh_user`` is
```

Comment:
Two backticks around `None` (you can change after tests pass and merge anyways).